subtests/docker_cli/run_env/run_env.py

```diff
@@ -3,493 +3,123 @@
 ---------
 
 This test checks function of `docker run -e VARIABLE=VALUE ...`.
-Also verifies the rm --link operation with ICC.
 
 Operational Summary
 ----------------------
 
-#. Setup client & server containers w/ exposed port and env. vars.
-#. Run code in containers to verify communication and env. vars
+#. Run container with -e options
 #. Check results match expectations
```

Comment:
Oh right, you are correct, and I might add as FYI, these subtest-module docstrings are what shows up in documentation for the test.  So the summary does tend to be very generic and high-level (for managers to read).  The detail you're suggesting I'd actually put into another (otherwise optional) section 'Operational Detail', that's intended to describe the individual sub-subtests more thoroughly.  I'll add in that section for what I've done here...